zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm

```diff
@@ -0,0 +1,120 @@
+//As a general rule, all origin backrounds must have summ of +5 of stat values
+
+/datum/category_item/setup_option/background/origin/oberth
+	stat_modifiers = list(
+		STAT_ROB = 5,
+		STAT_TGH = 5,
+		STAT_BIO = 0,
+		STAT_MEC = 0,
+		STAT_VIG = 10,
+		STAT_COG = -10
+	)
+
+
+/datum/category_item/setup_option/background/origin/predstraza
```

Comment:
Non-blocking -- can you add commentary describing what origins these are for future coders? While you can go digging for what 'predstraza' refers to, it's not known at a glance (at least for anyone unfamiliar with the language)